docker-compose.yml

```diff
@@ -12,16 +15,19 @@ services:
         restart: always
         command: webserver
```

Comment:
can we put the command in the Dockerfile?

Reply:
Done [here](https://github.com/datopian/ckan-ng-harvest/commit/ea0cadc8c44c99f8307900cf060b3186f3a3fc93)